### DISCUSSION TITLE
Should I use mean-var normalization for single user if stats_path is defined

### DISCUSSION THREAD

**Discussion by utunga** [coqui-ai/TTS#655] [General Q&A]
Hi! 

Just getting started training a model with Coqui TTS. I'm starting with just training  tacotron model to start - using train_tacotron.py script.  For the record I forked the repo - to add my own data preprocess setps at the `v0.0.14 
` tag. 

The pre-processing step correctly creates the stats and its a single speaker audio source so I feel like its approprirate to set the stats_path and set the signal_norm bool to true. 

config.json settings around normalization are like this:
<img width="739" alt="image" src="https://user-images.githubusercontent.com/166867/125429885-16e56328-3527-402a-a653-9456d93863f9.png">

However I ran into a problem when I had signal normalization enabled. So I had to set `signal_norm` to false in order to actually run a train. (Which works but is far from ideal)

Specifically this part of audio.py normalize() function would fail every time:

![image](https://user-images.githubusercontent.com/166867/125430547-32411a2e-1ed7-474f-a7e3-06d69e680028.png)

Basically for some reason the 0th axis of the sample data was 513 when the size_fft/2 was 512. 

I mention this just in case someone else has seen something similar??? Bit of a long shot, to be honest, but I figure it can't hurt to ask.

Or maybe I need upgrade to another version or something?

**Comment by utunga** [comment]
Thanks in advance for any comments - much appreciated!
